MachObfuscator/SymbolMangling/CezarMangler/CaesarMangler.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+final class CaesarMangler: SymbolMangling {
+    // todo: Probably should be changed to 'caesar'. Would require readme.md update
```

Comment:
I agree, that typo puts a shame on me :) What about updating this key and readme.md in this PR?

Reply:
It's just a typo, no worries! will add it to readme :)